website/src/redux/reducers/ReducerConstants.jsx

```diff
@@ -64,5 +64,6 @@ export const ELEMENT_ATTRS = [ELEMENT_POSITION, ELEMENT_SIZE, ELEMENT_ROTATION,
 
```

Comment:
Can you update the big comment above to include orientation so we get a big picture of how the tree looks.